components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -13,14 +13,22 @@
 // limitations under the License.
 
 #import "MDCChipTextField.h"
+
 #import "MaterialChips.h"
+#import "MDCChipTextFieldScrollView.h"
+
+//static CGFloat const kChipsSpacing = 10.0f;
 
-@interface MDCChipTextField ()
+@interface MDCChipTextField () <MDCChipTextFieldScrollViewDataSource>
 
-@property(nonatomic, strong) UIView *chipsView;
-@property(nonatomic) CGFloat insetX;
-@property(nonatomic, strong) NSLayoutConstraint *leadingConstraint;
-@property(nonatomic, strong) NSMutableArray<MDCChipView *> *chips;
+@property (nonatomic, weak) MDCChipTextFieldScrollView *chipsContainerView;
```

Comment:
I think this should be strong not weak.

Reply:
Addressed in 6d8a1c1dca231edfe812462981d84d9f003d47fa